locksmith/scripts/approve_erc20_transfer

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+const ethers = require('ethers')
+const args = require('yargs').argv
+
+let provider = new ethers.providers.JsonRpcProvider(args.provider)
+let value = args.value
+let lockContractAddress = args.lockContractAddress
+let erc20ContractAddress = args.erc20ContractAddress
+let privateKey = args.privateKey
```

Comment:
Is this a file path? If so, I would rename the var. If not, this is probably as unwise as passing a password on the command-line from a security perspective.

Reply:
it is not a file path, it is the private key used to sign the transaction. I'm not sure that reading from a file would be materially better

Reply:
https://unix.stackexchange.com/questions/78734/why-shouldnt-someone-use-passwords-in-the-command-line

and 

https://serverfault.com/questions/592631/passing-a-private-key-to-scp-from-the-command-line-instead-of-a-file

provide some context as to why this is a potentially bad idea

Reply:
cool, i'll make some adjustments in a future PR. To be clear this is to be run on our machines, I am not sure that offloading to a file will help if the machine is compromised but will give it a go